test/new-e2e/tests/agent-platform/common/helper/unix.go

```diff
@@ -6,13 +6,9 @@
 // Package helper implement interfaces to get some information that can be OS specific
 package helper
 
-import "github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-platform/common"
-
 // Unix implement helper function for Unix distributions
 type Unix struct{}
 
-var _ common.Helper = (*Unix)(nil)
```

Comment:
This was there to explicitly check that the interface is implemented

Reply:
It creates an import cycle. with `test_client.go`.

Why is a check here needed? If the interface isn't implemented then errors are raised where it's used, in `install_script_test.go`.

Reply:
It's a good practice across go, you can find it in go source code and in our too, example https://github.com/golang/go/blob/b44f6378233ada888f0dc79e0ac56def4673d9ed/src/crypto/aes/aes_gcm.go#L43
More examples: https://github.com/search?q=repo%3Agolang%2Fgo%20var%20_&type=code

If there is an import cycle, we should probably fix it